### ISSUE TITLE
JAWS does not read labels for multiple comboboxes in Edge browser

### ISSUE THREAD

**Issue by subhajit9614** [FreedomScientific/VFO-standards-support#595]
### Summary

> JAWS does not read labels for multiple comboboxes when comboboxes are added in DOM one after another.
> 
> Example:
> 1. Go to https://subhajitdas.w3spaces.com/saved-from-Tryit-2021-11-15-d7dw6.html
> 2. Tab to dropdown and autocomplete.

### Expected result

> JAWS should read label of autocomplete(second combobox) when focussed in it.
### Actual result

> JAWS only reads the label text for the first combobox and ignores the label text for the other comboboxes when I tab to them. This only happens when there are comboboxes right after another. JAWS doesn't have a problem reading the label if there's another input type separating the comboboxes.

### Additional Information
This is same is also raised in stackoverflow
https://stackoverflow.com/questions/69996416/jaws-does-not-read-labels-for-multiple-comboboxes
JAWS 2018 didn't have any issue. Once switched to JAWS 2019, issue appeared.

#### JAWS version and build number
JAWS 2019.1912.203 ILM

 
#### Operating System and version
Windows 10 64 bit

#### Browser and version:
Microsoft Edge : 95.0.1020.40

**Comment by JAWS-test** [comment]
I can confirm the error. However, it only occurs with JAWS 2019 and not with JAWS 2020 or newer version. With JAWS 2019, not only the label is not output, but neither is the role.

**Comment by StephenBeckettTPGi** [comment]
Yes, I confirm it is fine with JAWS 2021 as well @subhajit9614. Unfortunately it requires an update of JAWS to a later version.

**Comment by subhajit9614** [comment]
> Yes, I confirm it is fine with JAWS 2021 as well @subhajit9614. Unfortunately it requires an update of JAWS to a later version.

There are

**Comment by subhajit9614** [comment]
> Yes, I confirm it is fine with JAWS 2021 as well @subhajit9614. Unfortunately it requires an update of JAWS to a later version.

There is some constrained in updating JAWS to higher version. We are hoping for a patch fix in JAWS 2019.

**Comment by subhajit9614** [comment]
Reopened
